api/version/version.go

```diff
@@ -44,6 +44,8 @@ var (
 	V3_6 = semver.Version{Major: 3, Minor: 6}
 	V3_7 = semver.Version{Major: 3, Minor: 7}
 	V4_0 = semver.Version{Major: 4, Minor: 0}
+
+	AllVersions = []semver.Version{V3_0, V3_1, V3_2, V3_3, V3_4, V3_5, V3_6, V3_7, V4_0}
```

Comment:
How about add a comment, this must be sorted in ascending order? 

Alternately, sort this in `minSupportedVersion`?